src/microbe_stage/ProcessStatistics.cs

```diff
@@ -154,6 +148,8 @@ public float CurrentSpeed
         }
     }
 
+    public bool Enabled => Process.SpeedMultiplier > 0.5f;
```

Comment:
Wouldn't it be better to show it enabled if the value is above 0? I think somewhere else there's also a similar threshold so unless they both use zero, I'd put the magic value in Constants to make sure a consistent value is used.